README.md

```diff
@@ -13,6 +13,7 @@
 [![Docker Pulls](https://img.shields.io/docker/pulls/absaoss/k8gb)](https://hub.docker.com/r/absaoss/k8gb)
 [![Artifact HUB](https://img.shields.io/endpoint?url=https://artifacthub.io/badge/repository/k8gb)](https://artifacthub.io/packages/search?repo=k8gb)
 [![doc.crds.dev](https://img.shields.io/badge/doc-crds-purple)](https://doc.crds.dev/github.com/absaoss/k8gb)
+[![FOSSA Status](https://app.fossa.com/api/projects/custom%2B162%2Fgithub.com%2FAbsaOSS%2Fk8gb.svg?type=shield)](https://app.fossa.com/projects/custom%2B162%2Fgithub.com%2FAbsaOSS%2Fk8gb?ref=badge_shield)
```

Comment:
@idvoretskyi Is https://app.fossa.com/projects/custom%2B162%2Fgithub.com%2FAbsaOSS%2Fk8gb?ref=badge_shield supposed to be publicly available?
Currently getting
![image](https://user-images.githubusercontent.com/518532/114715428-592dd180-9d33-11eb-9e0d-116292159fc8.png)

Reply:
@ytsarev had the same, but also received invite to fossa, after that I'm able to see the page

Reply:
@ytsarev it's public now, however, I've invited you all to FOSSA as well (check your inbox for an invite, or DM me via the CNCF Slack).

Reply:
👍